src/docs/getting_started.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Getting Started
+---
+
+# Getting started
+
+In this section, we provide a high level overview on how to get started with Eclipse Theia and link to respective sections to read.
+Eclipse Theia is a platform for building custom Cloud & Desktop IDEs and tools with modern web technologies. Eclipse Theia is not a tool itself, but there are many tools which are built upon Theia. Theia as a project provides a template tool called [Eclipse Blueprint](https://theia-ide.org/docs/blueprint_download/) to try Theia and get started with building your own tool. Please have a look at the project goals for more details!
+
+## Frequently asked questions
+
+If you are unsure what Theia is, how it compares to other technologies or how to migrate Eclipse-based tools to it, please have a look at [this FAQ for Eclipse Theia](https://eclipsesource.com/blogs/2019/12/24/eclipse-theia-ide-faq/).
```

Comment:
I cannot move the content of the blog post. The document you linked is very outdated. So my proposal would be to leave link as is. If there are contributions to the FAQ or a new FAQ is contributed, we should change this link to point to it.
The only alternative is to not have a link to an FAQ, but I believe it adds value here.

Reply:
I wouldn't move the content from the blog post, but rather not hardlink to it directly. We can add the blog post link to our FAQ page I linked (if its outdated we can update it), or a dedicated page on the website that others can contribute to. I'd like to avoid having our documentation point to blog posts written/maintained by one contributing company and give the possibility for others to contribute their own helpful information.

Reply:
OK, makes sense, I linked to the resource section instead.